### DISCUSSION TITLE
Problem running npx from next tutorial.

### DISCUSSION THREAD

**Discussion by drayilara** [vercel/next.js#48515] [Help] [answered]
### Summary

I am new to next, I am going through the learning section in the docs. Currently trying to start a next project but I am encountering an error.

When I run this 

```
npx create-next-app@latest next-blog
```

I get  this error:

```
Need to install the following packages:
  create-next-app@13.3.0
Ok to proceed? (y) y
npm ERR! code E404
npm ERR! 404 Not Found - GET https://packagecloud.io/github/git-lfs/npm/create-next-app/-/create-next-app-13.3.0.tgz
npm ERR! 404 
npm ERR! 404  'create-next-app@https://packagecloud.io/github/git-lfs/npm/create-next-app/-/create-next-app-13.3.0.tgz' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/sodiq/.npm/_logs/2023-04-18T05_16_37_290Z-debug-0.log

```

I have tried to use v12XXX of next but no luck. Please help.

### Additional information

_os: Linux(ubuntu 22)_

**Comment by curiousYi** [comment]
Try running this command, which is in the [docs](https://nextjs.org/learn/basics/create-nextjs-app/setup), 

`npx create-next-app@latest nextjs-blog --use-npm --example "https://github.com/vercel/next-learn/tree/master/basics/learn-starter"` 

This worked for me.

**Reply by drayilara** [reply]
Hi thanks for responding.

I have tried the exact same code but it didn't work. No, I am not being a VPN. 
Maybe I will switch to an older version of node and try again, I am currently running node v19××.

I will post an update here.

**Reply by ethan-grzeda** [reply]
When I run this command, I get an error telling me that the repository could not be found...

**Comment by icyJoseph** [comment]
Hi,

Did you try again? Or are you behind some kind of VPN? 

I ran the exact same command, got the same question and it worked fine.

**Reply by drayilara** [reply]
Thanks.

I haven't tried again. I am not behind a VPN. 

I will try again and post an update.

**Comment by drayilara** [comment, marked as answer]
UPDATE - SOLVED.

I am currently running node v19XX, when I read the log by npm it said next.js wasn't in the registry, so I switched to node v14XX and this solved it.

Most likely a problem with the compatibility of the npm version that comes bundled with node 19, but node 14 works fine.

Thanks everyone.

**Reply by icyJoseph** [reply]
Try to use Node 18 :) 14 might also cause issues ~ at least 16 should be ok.

**Reply by ethan-grzeda** [reply]
I tried doing the same thing with node version 18, and I get the same error as at the beginning. My specific version is 18.17.1, if that helps.
